fe/fe-core/src/main/java/com/starrocks/sql/spm/BaselinePlan.java

```diff
@@ -17,48 +17,62 @@
 import com.starrocks.common.io.Writable;
 
 import java.time.LocalDateTime;
+import java.util.List;
 import java.util.Objects;
 
 public class BaselinePlan implements Writable {
-    @SerializedName("id")
+    public static final String SOURCE_USER = "USER";
+
+    public static final String SOURCE_CAPTURE = "CAPTURE";
+
     private long id;
 
-    private final boolean isGlobal;
+    private boolean isGlobal = false;
+
+    private boolean isEnable = false;
```

Comment:
There is some ambiguity about what enable specifically represents.

Reply:
meanings the baseline plan is enable